### PR TITLE
Update 2024190305_조규상.txt

### PR DESCRIPTION
**Pull request by kwuventure**
about branch protection

### CODE REVIEW

**Comment by kwuventure** [comment]
successfully protected!

**Review by kwuventure — COMMENTED** [review]
okay but cant approve

File: 2024190305_조규상.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
-main branch protection 적용 후 수정
+main branch protection 적용 시  
+  
+Do not allow bypassing the above settings
+The above settings will apply to administrators and custom roles with the "bypass branch protections" permission.  
 
-review needed 설정 후 수정
\ No newline at end of file
+위의 항목을 놓치지 않고 체크바람.
```
[the remark this answers is not in the document]
Comment (kyusang98) [reply 2024190305_조규상.txt:6]: kwuventure(administrator) cannot approve the pull request it made by itself.
so i was invited as a teammate. so i'll review and appove it.

File: 2024190305_조규상.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
-main branch protection 적용 후 수정
+main branch protection 적용 시  
+
```
Comment (kyusang98) [inline 2024190305_조규상.txt:2]: hmm..

**Review by kyusang98 — APPROVED** [review]
I'll approve of it!

**Comment by kyusang98** [comment]
you couldnt appove it by yourself so i've approved of it instead.
i'll merge as a collaborator(teammate)

**Pull request merged**